backend/lcfs/db/seeders/test_seeder.py

```diff
@@ -24,8 +24,8 @@ async def seed_test():
             await seed_test_organizations(session)
             await seed_test_user_profiles(session)
             await seed_test_user_roles(session)
-            await seed_test_transactions(session)
-            await seed_test_transfers(session)
+            # await seed_test_transactions(session)
```

Comment:
I noticed some tests require these test data in the test database. I believe starting with clean tables for transactions and transfers is better. For example, in my tests, I must account for these existing records to determine the number of transactions visible to a specific entity. It would be better to include the necessary test data within the test files. Otherwise, we'll need to consider the test data for certain tests in all other tests. Could you please share your opinion on this? Thanks

Reply:
Ok I see, lets use this approach moving forward.

Reply:
Sure! Thank you for considering my idea!